### ISSUE TITLE
Add support for slow connection and rotate direction.

### ISSUE THREAD

**Issue by khanglu** [andrepolischuk/react-rotation#12]
Hey Andrey,

Great little library here. I've just added it to my project and customise it to support the following things:

1. My sets of image rotates from left to right, with the default library it would rotate in the opposite direction, so I modified it (it's more efficient than re arranging the images order).
2. In slow connections, some images are not loaded in time and would collapse if you rotate into it. I modified the code to support this as well by using promises and Promises.all all the onLoad methods as well as displaying a little spinner when it's loading.

I can make a pull request on those if you're keen.

**Comment by andrepolischuk** [comment]
Hi, Im very sorry. I forgot about your issue.

1. Its great idea and I'll be glad if you make PR.
2. I think `react-rotation` not about deffered loading, and developers should handle loading and show spinner by himself in more specific components. For example:

```js
<Rotation>
  <LazyImg src='images/00.jpg' />
  <LazyImg src='images/01.jpg' />
  <LazyImg src='images/02.jpg' />
</Rotation>
```

**Comment by khanglu** [comment]
No problem at all, thanks for the reply! I was reading the points I made and realized it's not really necessary that's why I closed the issue.

I agree that adding loading indicator is not necessary. I'll make a PR later on the rotation direction.

**Comment by andrepolischuk** [comment]
Closed by #13
